sdk/react/utils.js

```diff
@@ -9,10 +9,24 @@ var {
   StyleSheet
 } = React;
 
-
 var Log = require('./log');
 var Utils = {
 
+  renderButton: function(style, icon, func, size, color, fontFamily) {
```

Comment:
good idea, @Matthew-C any objections to this?

If not, I would rename it to be more specific ( "renderRectButton")